client/css/style.css

```diff
@@ -1643,11 +1643,16 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	opacity: 1;
 }
 
-.textcomplete-item .emoji {
-	margin-right: 8px;
-	width: 16px;
-	text-align: center;
+.inline-emoji {
 	display: inline-block;
+	font-size: 1.4em;
+	vertical-align: bottom;
+	line-height: 1;
+}
+
+.textcomplete-item .inline-emoji {
```

Comment:
Kind of a nitpick but any reason to rename the class from `emoji` to `inline-emoji`? I'd rather keep the former if that's possible. :man_shrugging: